public/index.html

```diff
@@ -38,7 +38,9 @@
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
+    <header id="root-header"></header>
     <div id="root"></div>
+    <footer id="root-footer"></footer>
```

Comment:
Hablamos esto con @GusBaamonde.

El va a necesitar el Wrapper y esas clases para poder manipular estilos en los elementos por fuera de `root`.

```suggestion
    <div class="App dp-library dp-wrapper">
      <header id="root-header"></header>
      <div id="root"></div>
      <footer id="root-footer"></footer>
    </div>
```

Reply:
Usemos solamente `dp-library dp-wrapper` en el html y lo que se necesite ajustar en _App.tsx_ que se haga en un archivo _App.css_

Reply:
@GusBaamonde que opinás?

Reply:
Recién hablamos con @Cromeror sobre esto, yo no utilizo la clase App, eso ya estaba luego @Cromeror  hace una limpieza me dijo.

Reply:
🚀 

Y lo aplico en Swarm